backends/events/sdl/sdl-events.cpp

```diff
@@ -63,6 +70,31 @@ static uint32 convUTF8ToUTF32(const char *src) {
 
 	return utf32;
 }
+
+void SdlEventSource::loadGameControllerMappingFile() {
+	bool loaded = false;
+	if (ConfMan.hasKey("gcdbfile")) {
+		Common::FSNode file = Common::FSNode(ConfMan.get("gcdbfile"));
+		if (file.exists()) {
```

Comment:
If the user has configured a mapping file and it was not found, it would be nice to print a warning saying so.